### ISSUE TITLE
Ice is sublimated twice

### ISSUE THREAD

**Issue by rarutter** [ua-snap/dvm-dos-tem#366]
The value calculated for sublimation is subtracted from the snow layer ice content twice.

First in Snow_Env::updateDailySurfFlux()
```
    if (currl->isSnow && sublim>0.) {
        double layicermv = fmin(sublim, currl->ice);
        actsub += layicermv;
        sublim -= layicermv;
        **currl->ice -= layicermv;**
    }
```
And then again in Ground::constructSnowLayers()
```
    if(toplayer->isSnow) {
        toplayer->dz *= ((toplayer->ice -ablat)/(toplayer->ice));
        **toplayer->ice -= ablat;**
    }
```
layicermv and ablat are the same value in different scopes.

**Comment by rarutter** [comment]
The solution will be to remove the statement in updateDailySurfFlux. Implementing this will happen when we create a branch to fix the SNOWTHICK issue.

**Comment by rarutter** [comment]
Fixed by PR #372
